src/components/TravelClass.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PriceTypography from './PriceTypography';
+import './TravelClass.css';
+
+const TravelClassDetails = ({ travelClassName, availability }) => (
+  <div className="travelClass-leftGrid">
+    <div className="travelClass-name">{travelClassName}</div>
+    <div className="travelClass-availability">{availability}</div>
+  </div>
+
+);
+const TravelClassPrice = ({ fareDetails }) => (
+  <div className="travelClass-rightGrid">
+    <PriceTypography value={fareDetails?.price?.value} variant="caption" fontWeight={600} />
```

Comment:
You cannot use PriceTypography here; Again all the display related to pricing should be in <Price; Eg: Offer price, max_price etc.
think something like <Price variant="small" value={price}/>"
take inspiration from https://mui.com/material-ui/icons/#size

Reply:
updated